onnxscript/__init__.py

```diff
@@ -63,6 +63,12 @@
     # package is not installed
     pass
 
+def const (pyvalue: Any) -> Any:
```

Comment:
typing: T -> T?

Reply:
Sorry, this crept in accidentally into this PR. It was (meant to be) part of a different branch for an in-progress PR. Will remove it.